### PR TITLE
applications: nrf5340_audio: Using low latency setting as default

### PR DESCRIPTION
**Pull request by rick1082**
Using low latency configuration as default.
* Set transport latency to 5ms
* Set minimal presentation delay to 3ms
* Set prefer minimal presentation delay to 3ms 
OCT-2995
Signed-off-by: Jui-Chou Chung <jui-chou.chung@nordicsemi.no>

### CODE REVIEW

File: applications/nrf5340_audio/src/bluetooth/Kconfig [hunk 1]
```diff
@@ -82,7 +82,7 @@ config BT_AUDIO_PRESENTATION_DELAY_US
 config BT_AUDIO_MAX_TRANSPORT_LATENCY_MS
 	int "Max transport latency"
 	range 5 4000
-	default 20
+	default 5
```
[the remark this answers is not in the document]
Comment (rick1082) [reply applications/nrf5340_audio/src/bluetooth/Kconfig:85]: Yes, this can do some help. And the transport latency can be smaller than ISO interval.

**Review by alexsven — APPROVED** [review]

**Comment by rick1082** [comment]
> > This looks good, but we need to run the nightly latency tests before merging.
> 
> On the way, will share the result in here when I got it. But from my desk, the number will be CIS: 32ms BIS: 28ms The comparison will help us to tell the difference between audio analyzer and oscilloscope.

Now the test result shows
BIS: 30.79ms
CIS: 31.88ns

**Comment by koffes** [comment]
We need to add to the changelog as well. We may also discuss adding to known issues.

**Comment by rick1082** [comment]
Updated the MTL from 5ms to 10ms, the test result:
CIS: 33.55ms
BIS: 30.81ms

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -197,6 +197,7 @@ nRF5340 Audio
   * Removed the LE Audio controller for nRF5340 library.
     The only supported controller for LE Audio is :ref:`ug_ble_controller_softdevice`.
     This enables use of standard tools for building, configuring, and DFU.
+  * Using low latency configuration as default setting for the nRF5340 Audio application.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:200]: Suggesting that you organize the bullets under nRF5340 Audio as follows:

* Removed:

  * The LE Audio...

* Updated:

  * Low latency configuration to be used as default setting for the nRF5340 Audio application.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -192,12 +192,16 @@ Serial LTE modem
 nRF5340 Audio
 -------------
 
-* Updated:
+* Removed:
 
-  * Removed the LE Audio controller for nRF5340 library.
+  * The LE Audio controller for nRF5340 library is removed.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:197]:
```suggestion
  * The LE Audio controller for nRF5340 library.
```
The first bullet already says it is removed.

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**